### PR TITLE
TE5-T69 Swift-Basics

### PR DESCRIPTION
**Pull request by jay-a-shah**

### CODE REVIEW

File: Demo app/Demo app/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+//  Demo app
+//
+//  Created by Jay Shah on 11/01/22.
+//
+
+import UIKit
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
```
Comment (purvesh-d) [inline Demo app/Demo app/AppDelegate.swift:12]: Remove extra empty lines

File: Demo app/Demo app/SceneDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+//
+//  Created by Jay Shah on 11/01/22.
+//
+
+import UIKit
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+
+    var window: UIWindow?
+
```
Comment (purvesh-d) [inline Demo app/Demo app/SceneDelegate.swift]: Remove extra empty lines

File: Demo app/Demo app/ViewController.swift [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+
```
Comment (purvesh-d) [inline Demo app/Demo app/ViewController.swift]: Remove extra empty lines

File: Temporary.playground/Contents.swift [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import UIKit
+
+var greeting = "Hello, playground"
+func swap(_ a: inout Int, _ b: inout Int) {
+    let temporaryA = a
+    a = b
+    b = temporaryA
+}
```
Comment (purvesh-d) [inline Temporary.playground/Contents.swift]: Why this playground name is temporary? can't we give some meaningful name?

**Review by purvesh-d — CHANGES_REQUESTED** [review]

**Comment by purvesh-d** [comment]
- Remove extra empty lines from all the files.
-  Make single commits only.

File: Demo app/Demo app/AppDelegate.swift (hunk 1 above)
Comment (jay-a-shah) [reply Demo app/Demo app/AppDelegate.swift:12]: resolved

File: Demo app/Demo app/SceneDelegate.swift (hunk 2 above)
Comment (jay-a-shah) [reply Demo app/Demo app/SceneDelegate.swift]: okk I will Resolve It

File: Swift- Exercise.playground/Contents.swift [hunk 5]
```diff
@@ -0,0 +1,597 @@
...
+//Output: “Si”
+
+var str2 = "Simform"
+if str2.count > 2 {
+let str3 = str2.prefix(2)
+print("\(str3)")
+}else{
+     let str3 = str2.prefix(2)
+    print("Shorter String is \(str3)")
+
```
Comment (purvesh-d) [inline Swift- Exercise.playground/Contents.swift]: extra empty line

File: Swift- Exercise.playground/Contents.swift [hunk 6]
```diff
@@ -0,0 +1,597 @@
...
+print("Descending Order of this array is \(arr4.sorted(by: >))")
+
+//Print all the numbers from listOfNumbers that are divisible by at least one number from divisors
+//Input: var listOfNumbers = [1, 2, 3, 10, 100]
+//var divisors = [2, 5]
+//Output: 2,10,100
+
+var listOfNumbers = [1, 2, 3, 10, 100]
+var divisors = [2, 5]
+for i in listOfNumbers{
```
Comment (purvesh-d) [inline Swift- Exercise.playground/Contents.swift]: Code formating required

File: Swift-Basics .playground/Contents.swift [hunk 7]
```diff
@@ -0,0 +1,589 @@
...
+    let greeting = "Hello JavaTpoint " + "A platform for all technologies" + " \(name)"
+    return greeting
+}
+let result = newClosure("Hello Jay")
+print(result)
+
+var customersInLine = ["Cherry", "Aam", "Kiwi", "Apple", "Bannaa"]
+print(customersInLine.count)
+let customerProvide	 = {customersInLine.remove(at: 2)}
+print(customersInLine.count)
```
Comment (purvesh-d) [inline Swift-Basics .playground/Contents.swift:335]: Extra spaces

**Review by purvesh-d — CHANGES_REQUESTED** [review]

**Pull request merged**